src/renderer/components/FtTimestampCatcher.vue

```diff
@@ -1,25 +1,32 @@
 <template>
   <p
+    @click="onClick"
+    @keydown.space.prevent="onClick"
```

Comment:
**Non-blocking:** As this component is also used for every single comment under videos, where these event listeners would be unnecessary, it might make more sense to move them to the WatchVideoDescription component with the `.native` modifier added to them so that Vue applies them to the `<p>` element instead of the component instance.